tools/halide_image.h

```diff
@@ -184,6 +184,10 @@ class Image {
         return &(contents->buf);
     }
 
+    uint8_t *alloc() const {
+        return contents->alloc;
```

Comment:
I will drop alloc from the reported info & remove the method I added when I made this standalone.  I originally added this when we were chasing down alignment issues in the allocators...still somewhat interesting to see how often the allocation was already aligned to the desired amount.